site/content/en/docs/Reference/fleet.md

```diff
@@ -174,3 +174,41 @@ The `spec` field is the actual `FleetAllocation` specification and it is compose
   when the `FleetAllocation` is created
 - `metadata` is an optional list of custom labels and/or annotations that will be used to patch 
   the game server's metadata in the moment of allocation. This can be used to tell the server necessary session data
+
+{{% feature publishVersion="0.8.0" %}}
```

Comment:
Should we add a similar section to [Getting Started > Create a Fleet](https://github.com/GoogleCloudPlatform/agones/blob/master/site/content/en/docs/Getting%20Started/create-fleet.md)? and edit that section as well once this is out? (expiryVersion the old one, and pubishVersion the new?)